community/server/src/test/java/org/neo4j/server/rest/transactional/ExecutionResultSerializerTest.java

```diff
@@ -839,9 +925,23 @@ public Boolean answer( InvocationOnMock invocation ) throws Throwable
             when( executionResult.getExecutionPlanDescription() ).thenReturn( planDescription );
         }
 
+        when(executionResult.getNotifications()).thenReturn( notifications );
+
         return executionResult;
     }
 
+    private static Notification mockNotification(InputPosition position)
```

Comment:
You've got a perfectly good implementation class of `Notification`, there is no need to mock!